src/build/validator.md

```diff
@@ -1,7 +1,32 @@
 ---
-title: Validator Setup
+title: Validator Setup version 0.2
 ---
 
+## Version 0.2 updates 22/09/2021
+
+Find the full list of exact updates **22/09/2021**
+1. Add new section [How to separate your running nodes](/build/validator.html#how-to-separate-your-running-nodes)
+2. Update the parameter **PRIVATE_PEERS** in the step 6 within the section [Configure and start the Sentry node as a validator](/build/validator.html#configure-and-start-the-sentry-node-as-a-validator)
+3. Update the parameter **PRIVATE_PEERS** in the step 6 within the section [Configure and start the Seed node as a validator](/build/validator.html#configure-and-start-the-seed-node-as-a-validator)
+4. Update the [Ethereum full-node](/build/validator.html#ethereum-full-node) section with the [standard recommendation and specification](https://ethereum.org/en/developers/docs/nodes-and-clients/#recommended-specifications)
+5. Add new section [How to delete a current running node](/build/validator.html#how-to-delete-a-current-running-node)
+6. Update the section [hardware requirements](/build/validator.html#hardware-requirements)
+
+## How to delete a current running node
```

Comment:
Good to include this section, but I don't think we should open the page with instructions on how to delete a node, this could move down the page with a reference in the setup that links to it for anyone running the process for the second time